### ISSUE TITLE
Provide selection summary in select mode

### ISSUE THREAD

**Issue by namdre** [eclipse/sumo#9016]
It would be good to get a summary of selected objects in selection mode. Instead of the old count/lock panel  that is no longer needed after #8342. 
I propose a simple multi-lane text label that lists only those objects which are selected. I.e. "current selection:   edge:25, junction:3, busStop:1)

**Comment by palvarezlopez** [comment]
Fixed in #7884 (but invalid ticket reference)
